### ISSUE TITLE
apt with_items using deb throws a TypeError: argument of type 'NoneType' is not iterable on 2.0

### ISSUE THREAD

**Issue by egroeper** [ansible/ansible-modules-core#3277]
##### Issue Type:
- Bug Report
##### Plugin Name:

module: packaging/os/apt.py
##### Ansible Version:

```
ansible 2.1.0 (devel 2fa1936ff9) last updated 2016/03/13 20:09:29 (GMT +200)
```
##### Environment:

N/A
##### Summary:

ansible/ansible#14709 doesn't fix #2300 for deb

Running apt module with deb and with_items throws an exception:
TypeError: argument of type 'NoneType' is not iterable
##### Steps To Reproduce:

```
- name: some local debs
  apt: deb=/tmp/{{ item }}
  with_items: 
    - package1.deb
    - package2.deb
```
##### Expected Results:

Everything installed nicely. ;-)
##### Actual Results:

An exception occurred during task execution. To see the full traceback, use -vvv. The error was: TypeError: argument of type 'NoneType' is not iterable

There is also a note on the merged ansible/ansible#14709 regarding this issue.

I only opened a new issue, because to my knowledge it's hard to see comments on closed issues. Feel free to close this as duplicate of #2300 if you want to handle the issue there instead.

**Comment by bcoca** [comment]
can you please post the full traceback output?

**Comment by egroeper** [comment]
```
An exception occurred during task execution. The full traceback is:
Traceback (most recent call last):
  File "/usr/lib/pymodules/python2.7/ansible/executor/task_executor.py", line 92, in run
    item_results = self._run_loop(items)
  File "/usr/lib/pymodules/python2.7/ansible/executor/task_executor.py", line 223, in _run_loop
    items = self._squash_items(items, task_vars)
  File "/usr/lib/pymodules/python2.7/ansible/executor/task_executor.py", line 273, in _squash_items
    if templar._contains_vars(name):
  File "/usr/lib/pymodules/python2.7/ansible/template/__init__.py", line 373, in _contains_vars
    return self.environment.block_start_string in data or self.environment.variable_start_string in data
TypeError: argument of type 'NoneType' is not iterable

fatal: [node1]: FAILED! => {"failed": true, "msg": "Unexpected failure during module execution.", "stdout": ""}
```

**Comment by egroeper** [comment]
Oh sorry. It seems like your fix is simply not in the latest stable version available via the Ansible PPA for Ubuntu trusty (2.0.1.0-1ppa~trusty).

I will have a look, if the problem still exists in current devel.

**Comment by egroeper** [comment]
Everything runs fine on current devel branch.
